packages/snaps-sdk/src/jsx/validation.ts

```diff
@@ -341,7 +341,6 @@ export const AccountSelectorStruct: Describe<AccountSelectorElement> = element(
   'AccountSelector',
   {
     name: string(),
-    title: string(),
```

Comment:
Does the account selector have a static title now?

Reply:
I dropped this in favour of the usage of the `Field`'s `label` prop :)